content/faculty/honorboard.md

```diff
@@ -0,0 +1,75 @@
+title: Доска почёта
+slug: honorboard
+category: faculty
+tags: [_nav, _category_header, faculty]
+submenu_index: 4
+---
+
+<ul class="nav nav-tabs" id="myTab">
+   <li class="active"><a href="#tab1" data-toggle="tab">Сотрудники</a></li>
+  <li><a href="#tab2" data-toggle="tab">Обучающиеся</a></li>
+</ul>
+ 
+<div class="tab-content">
+  <div class="tab-pane active" id="tab1">
+  
+  
+<table class="honor_table">
+<tr>
+
+<td><img src="http://placehold.it/200x300">
+<br><b>Фамилия Имя Отчество</b>
```

Comment:
Using `<br>` tags instead of adding some more `<tr>` tags is a smell. Just a note. No need to rewrite.